tap_mailchimp/__init__.py

```diff
@@ -40,4 +40,4 @@ def main():
             sync(client,
```

Comment:
Add unit test for do_discover method

Reply:
Added unittest to cover do_discover and sync method.